apps/wing/src/cli.ts

```diff
@@ -36,6 +36,8 @@ async function main() {
     process.env.PROGRESS = "1";
```

Comment:
Since we should be able to get the logic of setting this env into one place, it may make sense to get rid of this and move it to the other function so that all the logic stays together.

That way, we can handle all three cases together: if progress is true, if it's false, and if it's undefined.